packages/read-emails/client/react/src/App.js

```diff
@@ -24,7 +24,7 @@ function App() {
           console.error('An error occurred parsing the response:', error);
         });
     }
-  }, [nylas]);
+  }, [nylas, userId]);
```

Comment:
Instead of this change, was this the missing code? https://github.com/nylas/use-cases/blob/d2b67bf73b5565cf9a59da2842004970be051367/packages/send-and-read-emails/client/react/src/App.js#L30
```
useEffect(() => {
    if (userId.length) {
      window.history.replaceState({}, '', `/?userId=${userId}`);
    } else {
      window.history.replaceState({}, '', '/');
    }
  }, [userId]);
  ```

Reply:
I can definitely change it to this!